include/utilities/realtime/dataman/DataMan.h

```diff
@@ -0,0 +1,468 @@
+#ifndef DATAMAN_H_
+#define DATAMAN_H_
+
+#include <chrono>
+#include <cstdint>
+#include <dlfcn.h>
+#include <functional>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <unistd.h>
+#include <vector>
+
+#include "external/json.hpp"
+
+using json = nlohmann::json;
```

Comment:
Move the `using json ...` into the class definition to prevent it from polluting other classes when included